packages/react-composites/src/composites/CallComposite/components/buttons/Microphone.tsx

```diff
@@ -41,8 +41,7 @@ export const Microphone = (props: {
    * When call is in Lobby, microphone button should be disabled.
    * This is due to to headless limitation where a call can not be muted/unmuted in lobby.
    */
-  if (_isInLobbyOrConnecting(callStatus)) {
-    microphoneButtonProps.disabled = true;
+  if (callStatus === 'Connecting') {
     // Lobby page should show the microphone status that was set on the local preview/configuration
     // page until the user successfully joins the call.
     microphoneButtonProps.checked = isLocalMicrophoneEnabled;
```

Comment:
Not related to this PR but `getLocalMicrophoneEnabled` selector should be changed to `getLocalMicrophoneOn`. The verb enabled makes it sound like it should decide if the button is enabled/disabled.